File: frontend/src/model/TrainingPlan.js
export function TrainingPlan(number, title, trainingType,
    personalTrainingsNumber, trainer, price, modifiedBy, createdBy,
    modificationDate, creationDate) {

  this.number = number;
  this.title = title;
  this.trainingType = trainingType;
  this.personalTrainingsNumber = personalTrainingsNumber;
  this.trainer = trainer;
  this.price = price;
  this.modifiedBy = modifiedBy;
  this.createdBy = createdBy;
  this.creationDate = creationDate;
  this.modificationDate = modificationDate;
}

File: backend/core/domain/src/main/java/pl/lodz/p/it/core/domain/TrainingPlan.java
package pl.lodz.p.it.core.domain;

import lombok.Data;
import lombok.EqualsAndHashCode;
import org.hibernate.validator.constraints.Range;
import pl.lodz.p.it.core.shared.validation.TrainingPlanNumber;

/**
 * Class responsible for keeping a domain model of the training plan object.
 */
@EqualsAndHashCode(callSuper = true)
@Data
public class TrainingPlan extends BaseModel {

    @TrainingPlanNumber
    private String number;

    private String name;

    private String trainingType;

    @Range(min = 0)
    private Integer personalTrainingsNumber;

    private String trainer;

    @Range(min = 1)
    private Double price;

    private String modifiedBy;

    private String createdBy;
}
